# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mesa/hotfix.patch.musl
# Copyright (C) 2016 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mesa-13.0.1/src/gallium/winsys/svga/drm/vmw_screen.h.vanilla	2016-12-24 15:02:06.967733193 +0000
+++ mesa-13.0.1/src/gallium/winsys/svga/drm/vmw_screen.h	2016-12-24 15:06:13.919737433 +0000
@@ -34,6 +34,7 @@
 #ifndef VMW_SCREEN_H_
 #define VMW_SCREEN_H_
 
+typedef unsigned long dev_t;
 
 #include "pipe/p_compiler.h"
 #include "pipe/p_state.h"
